garak/generators/openai.py

```diff
@@ -141,13 +142,16 @@ def __setstate__(self, d) -> object:
         self._load_client()
 
     def _load_client(self):
-        # Required stub implemented when extending `OpenAICompatible`
-        # should populate self.generator with an openai api compliant object
-        raise NotImplementedError
+        self.client = openai.OpenAI(base_url=self.uri, api_key=self.api_key)
```

Comment:
leave guidance for those implementing child classes
```suggestion
        # Required stub implemented when extending `OpenAICompatible`
        # should populate self.generator with an openai api compliant object
        self.client = openai.OpenAI(base_url=self.uri, api_key=self.api_key)
```

Reply:
There is not `requirement` to stub this method in extending classes, however I do like the idea of offering extra guidance.
```suggestion
        # When extending `OpenAICompatible` this method is a likely location for target application specific 
        # customization and must populate self.generator with an openai api compliant object
        self.client = openai.OpenAI(base_url=self.uri, api_key=self.api_key)
```